srcpkgs/python3-mysqlclient/template

```diff
@@ -10,6 +10,6 @@ depends="python3"
 short_desc="Python3 interface to MySQL/MariaDB"
 maintainer="Alin Dobre <alin.dobre@outlook.com>"
 license="GPL-2.0-or-later"
-homepage="https://github.com/PyMySQL/mysqlclient-python"
-distfiles="${PYPI_SITE}/m/mysqlclient/mysqlclient-${version}.tar.gz"
-checksum=3981ae9ce545901a36a8b7aed76ed02960a429f75dc53b7ad77fb2f9ab7cd56b
+homepage="https://github.com/PyMySQL/mysqlclient"
+distfiles="https://github.com/PyMySQL/mysqlclient/archive/refs/tags/v${version}.tar.gz"
```

Comment:
```suggestion
homepage="https://github.com/PyMySQL/mysqlclient"
changelog="https://raw.githubusercontent.com/PyMySQL/mysqlclient/main/HISTORY.rst"
distfiles="https://github.com/PyMySQL/mysqlclient/archive/refs/tags/v${version}.tar.gz"
```
why did you decide to switch to a github tarball?

Reply:
It seemed neater.

Reply:
If it isn't objectively better (for example containing tests), I think we should just use the pypi tarball.

Reply:
I have changed it back to the pypi tarball.